src/main/io/gps.c

```diff
@@ -417,9 +420,32 @@ void gpsInitNmea(void)
                if (!atgmRestartDone) {
                    atgmRestartDone = true;
                    serialPrint(gpsPort, "$PCAS02,100*1E\r\n");  // 10Hz refresh rate
```

Comment:
This will prevent user refresh rate unless it is last

Reply:
It's wrong everywhere but I think a different PR would be better to fix these.